### DISCUSSION TITLE
multidimensional NetCDF timeseries chart

### DISCUSSION THREAD

**Discussion by davidedelerma** [TerriaJS/terriajs#4932] [Q&A] [answered]
Hello, I have a multidimensional netcdf served with thredds and one of the dimensions is `elevation`
With the subsetting module I can get a chart of the time series when I click to a specific point on the map. 
I would like to parametrize also the elvation in my FeatureInfoTemplate using th mustache js templating, an example is below:
```json
name: 'NCEP 2019'
        type: "group"
        preserveOrder: true
        isOpen: false
        items:
        - layers: air
          elevation: 600
          name: "NCEP air temperature 2019 elevation 600 mbar"
          colorScaleMinimum: 178
          colorScaleMaximum: 320
          styles: "boxfill/red-yellow"
          type: "wms"
          url: "http://thredds:8080/thredds/wms/ncep/NCEP.air.2019.nc?service=WMS&version=1.3.0&request=GetCapabilities"
          featureInfoTemplate: '<p>NCEP air temperature elevation 600 mbar</p><chart src="http://thredds:8080/thredds/ncss/ncep/NCEP.air.2019.nc?var=air&latitude={{terria.coords.latitude}}&longitude={{terria.coords.longitude}}&time_start=2019-01-01T00:00:00Z&time_end=2019-11-01T00:00:00Z&vertCoord={{dimensions.elevation}}&accept=csv" y-column="4" hide-buttons="True"></chart>'
```
where instead of hardcoding vertCoord I try to pass it in the template as `vertCoord={{dimensions.elevation}}`, this solution is not working at the moment, is it possible to have something like that? 
If yes, could you help me to know which is the right parameter to pass?

Thanks

**Comment by tephenavies** [comment, marked as answer]
If the elevation is returned in the GetFeatureInfo response (in "properties" on the returned feature(s)) then you can use the name of the property in mustache. 

Here's a demo: http://ci.terria.io/master/#clean&https://gist.githubusercontent.com/steve9164/763db5aad1b8c926d3f90a0af222ec38/raw/a5ddee7642d7c84e7f744e75da7c480d0565324d/mustache-wms-catalog.json

Try clicking regions and check the feature info, the table you get when you click "Show Raw Data", and the response data when you inspect the network response for the GetFeatureInfo request clicking on a region (use network tab of browser debugger and filter by GetFeatureInfo).

**Reply by davidedelerma** [reply]
Awesome thanks

**Comment by rsignell-usgs** [comment]
@davidedelerma , did you get this to work, and if so, it is possible to access a demo?   
We would also love to see time series from our 4D model output on THREDDS!

**Reply by davidedelerma** [reply]
Unfortunately we had to split the netcdf file in different files, one per elevation :( Hopefully in terria8 there would be better support for timeseries from THREDDS...
